tests_e2e/orchestrator/lib/agent_test_suite.py

```diff
@@ -209,6 +211,7 @@ def _initialize(self, environment: Environment, variables: Dict[str, Any], lisa_
         self._environment_name = variables["c_env_name"]
 
         self._test_suites = variables["c_test_suites"]
+        self._test_args = variables["test_args"]
```

Comment:
looks like there may be inconsistency in type here. variables["test_argd"] is str while self._test_args is dict

Reply:
It's still a string, later we convert to dict. The variables exposed to user in pipeline, runbook or command line is in string in the form of list of comma-separated key=value pair
Example
`publihsedVersion=2.10.0.8,hostName=Test`

Reply:
oh, you are right. It should be string. I just noticed